templates/prWaitingRoom.html

```diff
@@ -21,11 +21,11 @@
     <script defer src="/__/firebase/init.js"></script> 
 
      <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
-    <link rel="stylesheet" type="text/css" href="prWaitingRoom.css">
-     <link rel="stylesheet" type="text/css" href="/temp/assets/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/font-awesome.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/templatemo-art-factory.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/owl-carousel.css">
+    <link rel="stylesheet" type="text/css" href="../temp/assets/css/prWaitingRoom.css">
```

Comment:
We've agreed to use the project directory as root for all hrefs. Change all of these hrefs to match this new naming system. You no longer need the escape sequence ".."
For example: 
https://github.com/mellychat/webapp/blob/6ee49c0ae171062c6f67f7218d7094988652a327/index.html#L17